tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -42,59 +43,65 @@ def generate_data(self, frames: list[dict[str, str | bool]], platform: str = "")
         test_data = {"platform": platform or self.platform, "stacktrace": {"frames": frames}}
         return self.store_event(data=test_data, project_id=self.project.id).data
 
+    def assert_slo_metric(self, mock_record, outcome=EventLifecycleOutcome.FAILURE, error_msg=None):
+        (events,) = (call for call in mock_record.mock_calls if call.args[0] == outcome)
+        if error_msg:
+            assert events.args[1] == error_msg
 
+
+@patch("sentry.integrations.utils.metrics.EventLifecycle.record_event")
 class TestTaskBehavior(BaseDeriveCodeMappings):
     """Test task behavior that is not language specific."""
 
     def setUp(self):
         super().setUp()
-        self.platform = "any"
         self.event_data = self.generate_data(
             [{"filename": "foo.py", "in_app": True}],
-            platform="any",
+            platform="javascript",
         )
 
-    def test_does_not_raise_installation_removed(self):
+    def test_does_not_raise_installation_removed(self, mock_record):
```

Comment:
this test wasn't testing anything because the platform "any" was not in the list of `SUPPORTED_LANGUAGES`. fixed that above